src/components/common/Metadata.jsx

```diff
@@ -8,8 +8,11 @@ import {
     Loader,
     Dropdown
 } from 'semantic-ui-react';
-import { BackendAPI } from '../common/API';
 import ReactGA from 'react-ga';
+import { fileDownload } from 'js-file-download';
```

Comment:
```suggestion
import fileDownload from 'js-file-download';
```
`fileDownload` is the default exported function and doesn't import and work correctly with the curly braces.

Reply:
Thanks!